### PR TITLE
Includes gatsby template

### PR DESCRIPTION
**Pull request by otaviojava**
This template has the plain template, to next step, I'll need to create documentation to integrate to drupal and thoese databases:

https://www.gatsbyjs.org/docs/sourcing-from-databases/

Ref: https://github.com/platformsh-templates/gatsby/pull/1

### CODE REVIEW

File: templates/gatsby/.platform.template.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# Once provisioned this file has no effects on the running project.  You
+# are free to remove it.
+
+# The schema is versioned so that we can establish code paths differently in the future if we need to change this.
+version: 1
+
+# Templates are a small amount of information supporting a template URL. Each template is selectable at the project-creation step.
+info:
+  # Unique machine name, prefaced by a vendor or organization identifier
+  id: platformsh/Gatsby
```
Comment (chadwcarlson) [inline templates/gatsby/.platform.template.yaml]:
```suggestion
  id: platformsh/gatsby
```

File: templates/gatsby/.platform.template.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# project creation time. This is part of the full v2 UI operation mode, which
+# places project schema/options selection early in the creation process, rather
+# than later as it exitss now. To allow this schema to be backwards-compatible,
+# this key also gets mapped to the appropriate location in project.settings so
+# that the current UI can have its own workflow overridden as well.
+initialize:
+  repository: git://github.com/platformsh-templates/gatsby.git@master
+  config: null
+  files: []
+  profile: Node.js
```
Comment (chadwcarlson) [inline templates/gatsby/.platform.template.yaml]:
```suggestion
  profile: Gatsby
```

File: templates/gatsby/files/.platform.app.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# .platform.app.yaml
+
+# The name of this application, which must be unique within a project.
+name: 'gatsbyjs'
```
Comment (chadwcarlson) [inline templates/gatsby/files/.platform.app.yaml]:
```suggestion
name: 'app'
```

File: templates/gatsby/files/.platform/routes.yaml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+"https://{default}/":
+  type: upstream
+  upstream: "gatsbyjs:http"
```
Comment (chadwcarlson) [inline templates/gatsby/files/.platform/routes.yaml]:
```suggestion
  upstream: "app:http"
```

File: templates/gatsby/files/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Gatsby for Platform.sh
+
+<p align="center">
+<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/gatsby/.platform.template.yaml&utm_content=gatsby&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
+    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
+</a>
+</p>
+
+This template builds a simple application using Gatsby.
+Gatsby is a free and open source framework based on React that helps developers build blazing fast websites and apps.
```
Comment (chadwcarlson) [inline templates/gatsby/files/README.md]:
```suggestion

Gatsby is a free and open source framework based on React that helps developers build blazing fast websites and apps.
```

Comment (Crell) [reply templates/gatsby/files/README.md]: This is from their marketing I assume?  It sounds too markety for our descriptions.

**Comment by otaviojava** [comment]
@Crell I got the information from their page.

**Pull request merged**